python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -135,9 +135,14 @@ def checkHostName():
                 self.syscfg.svo.disableService("tomcat6")
         except:
             pass
-            
+
         self.syscfg.svo.stopService("cloudstack-management")
-        if self.syscfg.svo.enableService("cloudstack-management"):
-            return True
+
+        if self.syscfg.env.noStart == False:
+            if self.syscfg.svo.enableService("cloudstack-management"):
+                return True
+            else:
+                raise CloudRuntimeException("Failed to configure %s, please see the /var/log/cloudstack/management/setupManagement.log for detail"%self.serviceName)
```

Comment:
Sorry to be a nag @ProjectMoon but can you change the path in the log message as well?